web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -49,563 +42,570 @@ import {FAIL, SUCCESS, ERROR, PENDING, RUNNING, HTTP_UNAUTHORIZED, LOGIN_URI, TE
+                                                    "" : <h2>Failed Tests</h2>;
+                                                isFailedTestsTitleAdded = true;
+                                                return (
+                                                    <div>
+                                                        {failedTestTitleContent}
+                                                        <h2 style={{color: "#e46226"}}>
+                                                            <a id={data.scenarioDescription}>
+                                                                {data.scenarioDescription}
+                                                                </a>
+                                                        </h2>
+                                                        <Table responsive>
+                                                            <thead displaySelectAll={false} adjustForCheckbox={false}>
+                                                            <tr>
+                                                                <th style={{width: "5%", textAlign: "center"}}/>
+                                                                <th style={{width: "30%"}}>Test Case</th>
+                                                                <th style={{width: "65%"}}>Failure Message</th>
+                                                            </tr>
+                                                            </thead>
+                                                            <tbody displayRowCheckbox={false}
+                                                                   showRowHover={true}>
+                                                            {data.testCaseEntries.map((entry, index) => {
+                                                                return (
+                                                                    <tr key={index}>
+                                                                        <td
+                                                                            style={{width: "5%"}}>
+                                                                            {entry.isTestSuccess ?
+                                                                                <img width="36"
+                                                                                     height="36"
+                                                                                     src={require('../success.png')}/>
+                                                                                :
+                                                                                <img width="36"
+                                                                                     height="36"
+                                                                                     src={require('../close.png')}/>}
```

Comment:
@lasanthaDLPDS As you see, four-space indentation tend to make the codes right-aligned. :) 
It's a bit harder to read as well.

Reply:
Fixed with 5c24430